coriolis/osmorphing/redhat.py

```diff
@@ -160,10 +202,25 @@ def _comment_keys_from_ifcfg_files(
                 "Commented all %s references from '%s'" % (
                     keys, fullpath))
 
+    def _get_os_version(self):
+        try:
+            version_str = self._detected_os_info.get('release_version', '0')
+            major_version = int(version_str.split('.')[0])
+            return major_version
+        except (ValueError, AttributeError):
+            LOG.warning(
+                "Could not parse OS version from detected_os_info: %s",
+                self._detected_os_info)
+            return
```

Comment:
Please return 0 here, as it's expecting to be an integer value when using this method.